src/Nancy/GlobalizationConfigurationExtensions.cs

```diff
@@ -13,10 +13,12 @@ public static class GlobalizationConfigurationExtensions
         /// </summary>
         /// <param name="environment">An <see cref="INancyEnvironment"/> that should be configured.</param>
         /// <param name="supportedCultureNames">Cultures that the application can accept</param>
-        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames)
+        /// <param name="defaultCulture">Used to set a default culture for the application <remarks>If not specified the first supported culture is used</remarks></param>
+        public static void Cultures(this INancyEnvironment environment, IEnumerable<string> supportedCultureNames, string defaultCulture = null)
         {
             environment.AddValue(new GlobalizationConfiguration(

```

Comment:
Actually not sure we need to use `First` as it will fall back to the Thread convention